util/annotations/annotations.go

```diff
@@ -283,3 +284,12 @@ func NewIncompatibleTypesInBinOpInfo(lhsType, operator, rhsType string, pos posr
 		Err:           fmt.Errorf("%w %q: %s %s %s", IncompatibleTypesInBinOpInfo, operator, lhsType, operator, rhsType),
 	}
 }
+
+// NewMixedFloatsHistogramsAggInfo is used when the queried series includes both
+// float samples and histogram samples in an aggregation.
+func NewMixedFloatsHistogramsAggInfo(aggregation string, pos posrange.PositionRange) error {
```

Comment:
I don't think this is correctly named and documented. This annotation is also returned if you try to only aggregate histograms (i.e. no floats, so no mix) with an aggregation operator that doesn't like histograms.

How about this:

```suggestion
// NewHistogramIgnoredInAggregationInfo is used when a histogram is ignored by
// an aggregation operator that cannot handle histograms.
func NewHistogramIgnoredInAggregationInfo(aggregation string, pos posrange.PositionRange) error {
```

Reply:
Yeah, it sounds like a better way to handle the info for these cases. I'll implement this.